hack/generate-operator-bundle.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+#
+# Generate an operator bundle for publishing to OLM. Copies appropriate files
+# into a directory, and composes the ClusterServiceVersion which needs bits and
+# pieces of our rbac and deployment files.
+#
+# Usage ./hack/generate-operator-bundle.py OUTPUT_DIR GIT_HASH
+
+import datetime
+import os
+import sys
+import yaml
+import shutil
+
+PACKAGE = '''packageName: hive-operator
+channels:
+- name: alpha
+  currentCSV: %s
+'''
+
+if len(sys.argv) != 4:
+    print("USAGE: %s OUTPUT_DIR GIT_HASH HIVE_IMAGE" % sys.argv[0])
+    sys.exit(1)
+
+outdir = sys.argv[1]
+git_hash = sys.argv[2]
+hive_image = sys.argv[3]
+
+if not os.path.exists(outdir):
+    os.mkdir(outdir)
+
+# Copy all CSV files over to the bundle output dir:
+crd_files = os.listdir('config/crds/')
+for file_name in crd_files:
+    full_path = os.path.join('config/crds', file_name)
+    if (os.path.isfile(os.path.join('config/crds', file_name))):
+        shutil.copy(full_path, os.path.join(outdir, file_name))
+
+with open('config/templates/hive-csv-template.yaml', 'r') as stream:
+    csv = yaml.load(stream)
+
+csv['spec']['install']['spec']['clusterPermissions'] = []
+
+# Add our operator role to the CSV:
+with open('config/operator/operator_role.yaml', 'r') as stream:
```

Comment:
Just for future improvement ... wouldn't we want to add `config/operator/*_role.yaml` to the roles?

Reply:
I don't forsee more roles for the operator, but maybe we could move this one  alongside the others in config/rbac/, and then add config/rbac/*_role.yaml.